### PR TITLE
Added better support for troubleshooting email sending.

### LINKED ISSUE #1114: support better `asgs-sendmail` troubleshooting
- [x] add `$SCRIPTDIR/mail.log` support to `inspect` and  `edit` commands
- [x] add `$HOME/.asgsh_profile` support to `inspect` and  `edit` commands
- [x] add `$HOME/asgs-global.conf` support to `inspect` and  `edit` commands
- [x] add an interactive `verify sendmail` command
- [x] add `mail.log` support as `tailf mail-log`, update bash completion for `tailf`

### PR DESCRIPTION
**Pull request by wwlwpd**
Issue 1114:

* added `$SCRIPTDIR/mail.log` support to `inspect` and `edit` commands
* addedj `$HOME/.asgsh_profile` support to `inspect` and `edit` commands
* added `$HOME/asgs-global.conf` support to `inspect` and `edit` commands
* added an interactive verify sendmail command
* added `mail.log` support as `tailf mail-log`, update bash completion for `tailf`

Resolves #1114.
Resolves #1116 (PR).

### CODE REVIEW

File: bin/verify [hunk 1]
```diff
@@ -144,6 +145,19 @@ verify_email_config() {
...
       echo "ok, permissions fixed for $HOME/asgs-global.conf"
   fi
+  if [ -n "$TESTTO" ]; then
+    echo ++ Sending a test email to: $TESTTO
+    cat $(which asgs-sendmail) | asgs-sendmail --to $TESTTO --subject "ASGS email send verification test"
+    err=$?
+    if [ $? == 0 ]; then
+      echo "ok, no errors detected during send; check inbox for test email"
+    else
+      echo "not ok, sending email via asgs-sendmail resulted in an exit code of $err"
```
Comment (jasonfleming) [inline bin/verify:155]: This error message should probably be preceded with `warning:` to be consistent.

File: cloud/general/DOT-asgs-brew.sh [hunk 2]
```diff
@@ -556,17 +549,31 @@ edit() {
   adcirc)
     BRANCH=${2}
     if [ ! -e "$ADCIRC_META_DIR/$BRANCH" ]; then
-      echo "An ADCIRC environment named '$BRANCH' doesn't exist"
+      echo "$I An ADCIRC environment named '$BRANCH' doesn't exist"
```
Comment (jasonfleming) [inline cloud/general/DOT-asgs-brew.sh:552]: `warning:` or `error:`?

**Review by jasonfleming — APPROVED** [review]
These look great @wwlwpd thank you. I had one minor comment that is optional to address. We also need to update the all-important ASGS Shell Cheat Sheet with these new capabilities. Not sure if I need to log a new issue to track that but can do, just let me know.

**Comment by wwlwpd** [comment]
> These look great @wwlwpd thank you. I had one minor comment that is optional to address. We also need to update the all-important ASGS Shell Cheat Sheet with these new capabilities. Not sure if I need to log a new issue to track that but can do, just let me know.

I converted this comment into an issue, there are other things I need to add as well.

File: bin/verify (hunk 1 above)
Comment (wwlwpd) [reply bin/verify:155]: This is mimicking the "TAP" output of ok/not ok and is meant for parsing; thank you for pointing that out. I can explain next time we talk.

**Pull request merged**